src/CSharp/CodeCracker/Refactoring/ChangeAsToCastAnalyzer.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections.Immutable;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace CodeCracker.CSharp.Refactoring
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public class ChangeAsToCastAnalyzer : DiagnosticAnalyzer
+    {
+        internal const string Title = "Toggle between safe cast ('as') and direct cast.";
+        internal const string MessageFormat = "Toggle between safe cast ('as') and direct cast.";
```

Comment:
Use resources for these strings. See the `NameOfAnalyzer` with the use of `LocalizableResourceString`.